src/test/java/com/ioet/bpm/people/boundaries/PeopleControllerTest.java

```diff
@@ -110,12 +116,13 @@ public void ifThePersonToUpdateDoesNotExistA404IsReturned() {
     }
 
     @Test
-    public void whenAPersonIsUpdatedTheUpdatedPersonIsReturned() {
-
+    public void whenAPersonIsUpdatedTheUpdatedPersonIsReturned() throws UnsupportedEncodingException, NoSuchAlgorithmException {
```

Comment:
same here with exceptions.

Reply:
Isn't this weird?

        person.setCreated(new Timestamp(System.currentTimeMillis()));
        person.setUpdated(new Timestamp(System.currentTimeMillis()));

Is this a new person or a new password which is being "created"?

Wouldn't we expect to see the first updated timestamp to be = to the created timestamp? I mean after an object has been created of course many updates can come after in time sequence; but would think that state set at creation time would be set "at creation time"; not a few nanoseconds later.

Reply:
yup